index.html

```diff
@@ -139,26 +146,26 @@ <h2>Profesional Art Printing Data</h2>
                 <li><a href="#">Bootstrap</a></li>
                 <li><a href="#">RUBY</a></li>
             </ul>
-            <a class="see-project" href="#">See project</a>
+            <a class="see-project none" href="#">See project</a>
         </div>
     </section>
     <section class="about-myself">
-        <h2>About me</h2>
-        <p>
-            Hello I'm a software developer! I can help you build a product ,
-            feature or website Look through some of my work and experience!
-            If you like what you see and have a project you need coded, don't hesitate to contact me.
-        </p>
-        <a class="see-project button-1" href="#">Get my resume</a>
-        <!-- <img src="images/Frame.svg" alt="frame image"> -->
-
-        <div class="img-get" aria-hidden="true"></div>
-        <hr>
+        <div id="about-info">
+            <h2>About me</h2>
+            <p>
+                Hello I'm a software developer! I can help you build a product ,
+                feature or website Look through some of my work and experience!
+                If you like what you see and have a project you need coded, don't hesitate to contact me.
+            </p>
+            <a class="see-project button-1" href="#">Get my resume</a>    
+        </div>
+       <img id="frame" src="images/illustration about me.svg" alt="frame image">
+        <hr id="line-1">
+        <hr id="line-2">
         <div class="categories">
-            <ul class="outer-list">
-                <li>
+                <div class="outer-list">
                     <div class="footer-heading font">
-                        <img src="images/Rectangle.svg" alt="Rotated rectangle">
+                        <img src="images/rectangle.svg" alt="Rotated rectangle">
```

Comment:
- [ ] Hello @ClarenceAtim, please fix the image not displaying issue as requested by the previous reviewers in order to adhere to the Figma design.

I think the right name of the image is `Rectangle.svg` instead of `rectangle.svg`.

```suggestion
                        <img src="images/Rectangle.svg" alt="Rotated rectangle">
```